tests/phpunit/includes/abstract-testcase.php

```diff
@@ -1532,6 +1532,23 @@ public function _make_attachment( $upload, $parent_post_id = 0 ) {
 		return $id;
 	}
 
+	/**
+	 * Uploads given file and creates an attachment post from it.
+	 *
+	 * @since 6.1.0
```

Comment:
```suggestion
	 * @since 6.2.0
```

Update since to current milestone version.

Reply:
Updated to 6.2.0.